### ISSUE TITLE
Grid missing option to overwrite "aria-label"

### ISSUE THREAD

**Issue by slafleche** [bvaughn/react-virtualized#1068]
It's good to have a property to overwrite the "role" for the Grid, but we don't have an property to change the "aria-label".

**Comment by bvaughn** [comment]
There is an `aria-label` prop :)
